src/ruleLoader.ts

```diff
@@ -57,26 +57,40 @@ export function findRule(name: string, rulesDirectory?: string) {
         return Rule;
     }
 
-    // then check for rules within the first level of rulesDirectory
-    if (rulesDirectory) {
-        Rule = loadRule(rulesDirectory, camelizedName);
-        if (Rule) {
-            return Rule;
+    // normalize the rulesDirectories argument
+    let directories: Array<string> = [];
+
+    if (rulesDirectories) {
+        if (typeof rulesDirectories === "string") {
+            directories.push(rulesDirectories);
+        } else {
+            directories.push(...rulesDirectories);
         }
     }
 
-    // finally check for rules within the first level of directories,
-    // using dash prefixes as the sub-directory names
-    if (rulesDirectory) {
-        const subDirectory = strLeft(rulesDirectory, "-");
-        const ruleName = strRight(rulesDirectory, "-");
-        if (subDirectory !== rulesDirectory && ruleName !== rulesDirectory) {
-            camelizedName = transformName(ruleName);
-            Rule = loadRule(rulesDirectory, subDirectory, camelizedName);
+    // loop through the custom directories
+    for (let rulesDirectory of directories) {
+        // then check for rules within the first level of rulesDirectory
+        if (rulesDirectory) {
+            Rule = loadRule(rulesDirectory, camelizedName);
             if (Rule) {
                 return Rule;
             }
         }
+
+        // finally check for rules within the first level of directories,
+        // using dash prefixes as the sub-directory names
+        if (rulesDirectory) {
+            const subDirectory = strLeft(rulesDirectory, "-");
+            const ruleName = strRight(rulesDirectory, "-");
+            if (subDirectory !== rulesDirectory && ruleName !== rulesDirectory) {
+                camelizedName = transformName(ruleName);
+                Rule = loadRule(rulesDirectory, subDirectory, camelizedName);
+                if (Rule) {
```

Comment:
Although it was not my code, I made the changes in commit https://github.com/palantir/tslint/commit/ee60a0cd1b858bd9555f21e9b4efdb501ebb570d.